compiler/rustc_infer/src/infer/region_constraints/mod.rs

```diff
@@ -256,7 +262,15 @@ pub enum VerifyBound<'tcx> {
 /// for cases like
 ///
 /// ```rust
-/// where for<'a> <T as SomeTrait<'a>::Item: 'a
+/// # trait SomeTrait<'a> {
+/// #    type Item;
+/// # }
+/// fn foo<'a, 'b, T: SomeTrait<'a>>()
+/// where
+///     <T as SomeTrait<'a>>::Item: 'a
+/// {
+///
+/// }
```

Comment:
This is a lot more verbose and makes it harder to understand that the where clause is what's relevant here. I think it makes sense to `ignore` this block.